### PR TITLE
chore(ctp): Update ctb dep version

### PR DESCRIPTION
**Pull request by maurelian**
The release action is failing due to a mismatch between the actual contracts-bedrock package version, and the version used in the periphery package. 

This change was made in #4597 to 'fix an error in CI', so hopefully this reversal of the change doesn't cause a new error, or we may find ourselves in a stalemate between CI and the release process. 😳

### CODE REVIEW

File: packages/contracts-bedrock/foundry.toml [hunk 1]
```diff
@@ -7,7 +7,6 @@ optimizer_runs = 999999
 remappings = [
   '@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable/',
   '@openzeppelin/contracts/=node_modules/@openzeppelin/contracts/',
-  '@eth-optimism/contracts-periphery/=node_modules/@eth-optimism/contracts-periphery/contracts',
```
Comment (maurelian) [inline packages/contracts-bedrock/foundry.toml:10]: This was wrong (should remap to `../../@eth-optimism`), but it doesn't appear to be in use so I've just removed it.

Comment (tynes) [reply packages/contracts-bedrock/foundry.toml:10]: you cannot import remappings outside of the current dir, the base layer `package.json` hoists `contracts-periphery` into the local node modules here, if `contracts-periphery` is no longer a dep to `contracts-bedrock` then we can delete the hoisting in the base `package.json`

Comment (maurelian) [reply packages/contracts-bedrock/foundry.toml:10]: In the end it seems that: 
1. remappings outside the current dir (as above) do work
2. despite [this change](https://github.com/ethereum-optimism/optimism/pull/4617/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R28) (which I thought would properly apply `nohoist` to the `contracts-bedrock` package), those contracts are still found in the root `node_modules` dir. I have checked that this is the case both locally and by ssh'ing into CI. 

It all seems a bit odd, but given the checks are now passing I'm going to resolve which will hopefully unblock the release process.

File: packages/contracts-periphery/foundry.toml [hunk 2]
```diff
@@ -18,7 +18,7 @@ remappings = [
 	'ds-test/=node_modules/ds-test/src',
   '@openzeppelin/contracts/=node_modules/@openzeppelin/contracts/',
   '@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable/',
-  '@eth-optimism/contracts-bedrock/=node_modules/@eth-optimism/contracts-bedrock',
+  '@eth-optimism/contracts-bedrock/=../../node_modules/@eth-optimism/contracts-bedrock',
```
Comment (tynes) [inline packages/contracts-periphery/foundry.toml:21]: I am surprised that this builds? It used to not build like this

**Comment by tynes** [comment]
Curious what is different about weth9, guessing its the metadata hash

**Review by tynes — APPROVED** [review]

**Comment by maurelian** [comment]
> Curious what is different about weth9, guessing its the metadata hash

Good question. Do we include metadatahashes in the go bindings?

**Pull request merged**